app/controllers/book_suggestions_controller.rb

```diff
@@ -0,0 +1,14 @@
+class BookSuggestionsController < ApplicationController
+  def create
+    book_suggestion = BookSuggestion.create(book_suggestion_creation_params)
+    if book_suggestion.persisted?
+      render json: book_suggestion, serializer: BookSuggestionSerializer, status: :created
```

Comment:
It should use BookSuggestionSerializer by default. Rails constantly uses convention over configuration, so it assumes if your model is called Model that there will be a ModelSerializer (if not it'll use a default serializer). You only specify the serializer when you're using a particular serializer with a different name (like some PublicModelSerializer) or if it's an each_serializer. Check all controllers to see if I ignored this in my other PR corrections (sorry)

Reply:
Also, if you remove this you might be able to do 
```
return render json: book_suggestion, status: :created if book_suggestion.persisted?
render json: book_suggestion.errors.messages, status: :unprocessable_entity
```
(notice the return though! render doesn't return from the method on its own, and if you forget it you'll just get an error about calling the render method twice)